app/backend/approaches/AskData.py

```diff
@@ -0,0 +1,161 @@
+import streamlit as st
```

Comment:
All new files need the Copyright and License headers added please...

Reply:
Updated.